prow/config/secret/agent_test.go

```diff
@@ -82,7 +84,7 @@ func TestCensoringFormatter(t *testing.T) {
 		DisableColors:    true,
 		DisableTimestamp: true,
 	}
-	formatter := agent.GetCensoringFormatter(baseFormatter)
+	formatter := logrusutil.NewCensoringFormatter(baseFormatter, agent.getSecrets)
```

Comment:
The reason that I did not remove the tests here:
The secrets for SA are loaded in another go routine ... i feel it is safer to keep this this scenario.